llvm/test/CodeGen/AIE/aie2/schedule/postpipeliner/conv2d.mir

```diff
@@ -100,10 +100,10 @@
   ; CHECK-NEXT:    vldb wh6, [p0], m6; nopa ; nops ; nopxm ; nopv
   ; CHECK-NEXT:    vldb wl8, [p0], m6; nopa ; nops ; add r0, r5, #33; vshift.align x2, x2, s1, x8, r0; nopv
   ; CHECK-NEXT:    vldb.3d wh8, [p0], d0; nopa ; nops ; nopx ; vshuffle x9, x4, x2, r2; nopv
-  ; CHECK-NEXT:    vldb wl10, [p1], #32; nopa ; nops ; nopx ; vshuffle x3, x4, x2, r3; nopv
-  ; CHECK-NEXT:    vldb wh10, [p1], #32; nopx ; vshuffle x1, x9, x0, r8; vmac cm1, cm1, x9, x10, r4
-  ; CHECK-NEXT:    vldb wl7, [p1], #32; vshuffle x5, x3, x0, r8; vmac cm3, cm3, x3, x10, r4
-  ; CHECK-NEXT:    vldb wh7, [p1], #32; mov r6, p0; vmac cm2, cm2, x1, x10, r4
```

Comment:
This diff is a case of the new, simple heuristic added at the top finding the II that was found earlier by a more complex one. Different schedule, same II.